tests/run-make/core-ffi-typecheck-clang/rmake.rs

```diff
@@ -0,0 +1,194 @@
+//@ needs-force-clang-based-tests
```

Comment:
Nit (style): can you move this directive below, and change the `//` comment block below to become a `//!` doc comment block?